cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/inputparameters/r4/InputParameterResolver.java

```diff
@@ -115,44 +116,36 @@ protected boolean validateContext(SDC_QUESTIONNAIRE_LAUNCH_CONTEXT_CODE code, St
 
     protected void resolveLaunchContext(
             Parameters params,
-            List<? extends IBaseBackboneElement> contexts,
+            List<IParametersParameterComponentAdapter> contexts,
             List<IBaseExtension<?, ?>> launchContexts) {
         launchContexts.stream().map(e -> (Extension) e).forEach(launchContext -> {
             var name = ((Coding) launchContext.getExtensionByUrl("name").getValue()).getCode();
-            var type = launchContext
-                    .getExtensionByUrl("type")
-                    .getValueAsPrimitive()
-                    .getValueAsString();
-            if (!validateContext(SDC_QUESTIONNAIRE_LAUNCH_CONTEXT_CODE.valueOf(name.toUpperCase()), type)) {
-                throw new IllegalArgumentException(String.format("Unsupported launch context for %s: %s", name, type));
-            }
+            var types = launchContext.getExtensionsByUrl("type").stream()
+                    .map(type -> type.getValueAsPrimitive().getValueAsString())
+                    .collect(Collectors.toList());
             var content = getContent(contexts, name);
-            if (content == null || content.isEmpty()) {
-                throw new IllegalArgumentException(String.format("Missing content for context: %s", name));
-            }
-            var value = getValue(type, content);
-            if (!value.isEmpty()) {
-                var resource =
-                        (Resource) (value.size() == 1 ? value.get(0) : BundleHelper.newBundle(FhirVersionEnum.R4));
-                if (value.size() > 1) {
-                    value.forEach(
-                            v -> ((Bundle) resource).addEntry(new BundleEntryComponent().setResource((Resource) v)));
+            if (content != null && !content.isEmpty()) {
+                var value = getValue(types, content);
+                if (!value.isEmpty()) {
+                    var cqlParameterName = name.substring(0, 1).toUpperCase().concat(name.substring(1));
```

Comment:
This is weird:  are we always assuming the CQL parameter name is a single character?